src/Bones/bone.ts

```diff
@@ -244,7 +244,7 @@ export class Bone extends Node {
         if (this._skeleton._numBonesWithLinkedTransformNode > 0) {
             this.updateMatrix(this._restPose.clone(), false, false);
         } else {
-            this.updateMatrix(this._restPose.clone());
+            this.updateMatrix(this._restPose.clone(), false, true);

```

Comment:
I know this was already like that, but since you are already changing it - why use clone, if updateMatrix copies the content and doesn't change the matrix provided? This generates an unneeded object that is being GC'ed right after

Reply:
True, I have performed the change.